README.md

```diff
@@ -1,2 +1,72 @@
-# project-product-landing-page
-This project is because of project product landing page
+# Project Product Landing Page 🚀
+
+## Project Description 📝
+
+I create this project becuase of learning there was some problem i face to it first that i should insert some text inside html tag it should not be empty.
+I learn a lot things in this project special responsive the page an media query creating the logo and navbar.the simple design that i use it it make my project stand out 
+
+```html
+<footer>
+    <ul>
+      <li class="list">Contact Us:</li>
+      <li><a href=""><img src="access/5296499_fb_facebook_facebook logo_icon.png" class="img1"></a></li>
+      <li><a href="#"><img src="access/5296765_camera_instagram_instagram logo_icon.png" class="img1"></a></li>
+      <li><a href="#"><img src="access/5296520_bubble_chat_mobile_whatsapp_whatsapp logo_icon.png" class="img1"></a></li>
+    </ul>
+</footer>
+```
+
+```css
+@media (max-width: 800px) {
+  .product {
+    max-width: 300px;
+    width: 100%;
+    margin: 0 auto;
+    margin-bottom: 10px;
+  }
+}
+```
+## Demo 📸
+
+![Project Demo](access/Screenshot%20(189).png)
+
+## Technologies Used 🛠️
+- HTML
+- CSS
+
+## Installation 💻
+
+```clone
+  git@github.com:humuyar/project-product-landing-page.git
+```
+
+## Usage 🎯
+
+```clone
+  cd project-product-landing-page
+```
+```clone
+  code .
+```
+```clone
+  git add .
+```
+```clone
+  git commit -m "anything you want"
+```
+```clone
+  git push
+```
```

Comment:
- Remove this as this part is only for installation not for contribution

Reply:
ok sir